openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -666,6 +666,13 @@ const GlossaryPageV1 = () => {
     }
   };
 
+  const afterDeleteAction = () => {
+    setGlossariesList([]);
+    setIsLoading(true);
+    history.push(getGlossaryPath());
```

Comment:
Instead of always redirecting to Glossary path, can we check if deleted Glossary-Term has a parent, then that should be the active node after re-routing. 
In case of deleting Glossary, above redirection will work fine.

CC: @harshach @vivekratnavel

Reply:
Sure i Will check for that!